get.sh

```diff
@@ -73,19 +74,49 @@ if [ -z "$CODACY_REPORTER_VERSION" ]; then
     CODACY_REPORTER_VERSION="latest"
 fi
 
-codacy_reporter="$codacy_temp_folder/codacy-coverage-reporter-assembly.jar"
+download_coverage_reporter() {
+    if [ ! -f "$codacy_reporter" ]
+    then
+        log "$i" "Download the codacy reporter $1... ($CODACY_REPORTER_VERSION)"
+        curl -LS -o "$codacy_reporter" "$(curl -LSs https://api.github.com/repos/codacy/codacy-coverage-reporter/releases/$CODACY_REPORTER_VERSION | grep browser_download_url | grep $1 | cut -d '"' -f 4)"
```

Comment:
```suggestion
        curl -# -LS -o "$codacy_reporter" "$(curl -LSs https://api.github.com/repos/codacy/codacy-coverage-reporter/releases/$CODACY_REPORTER_VERSION | grep browser_download_url | grep $1 | cut -d '"' -f 4)"
```
this will output a one line progress bar `###...## 100.0%`

Reply:
Really nice suggestion, it looks a lot better with the progress bar 🙏👍